app/controllers/api/v2/hosts_controller.rb

```diff
@@ -238,6 +238,18 @@ def rebuild_config
         end
       end
 
+      api :GET, "/hosts/:id/template/:type", N_("Get host template")
+      param :id, :identifier_dottable, :required => true
+      param :type, String, :desc => N_("Template type")
```

Comment:
It's probably worth calling it a template *kind* too, which matches the other config template controller parameters.